virtool/analyses/data.py

```diff
@@ -96,7 +96,7 @@ async def find(
             {"$match": {"sample.id": sample_id} if sample_id is not None else {}},
             {
                 "$facet": {
-                    "total_count": [{"$count": "total_count"}],
```

Comment:
Please don't change how this is implemented versus `main`.

There may have been some confusion as a fix was merged to remediate very slow responses for this method in production. It turns out that `count_documents({})` is extremely slow for a collection of 1000s of large documents.